README.md

```diff
@@ -165,15 +165,15 @@ shown below. Can't see yours? Have a look at the [extra platform instructions](h
    ##### From prebuilt binary, with Shell:
 
    ```sh
-   sh -c "$(curl -fsSL https://starship.rs/install.sh)"
+   curl -fsSL https://starship.rs/install.sh | sh
```

Comment:
It looks like we changed this in #2553 to avoid potential attacks, is there some way to avoid timing attacks but still be compatible with fish?

Reply:
`sh -c 'sh -c "$(curl -fsSL https://starship.rs/install.sh)"'` would work.

Reply:
> sh -c 'sh -c "$(curl -fsSL https://starship.rs/install.sh)"'

Yes, it works in fish! 👍

Reply:
If I were to see that script, I'd probably think the repetition was a typo, if I'm honest.
 
I don't know that we should set the precedent making our init script support fish. It would probably lead to us making similar changes to scripts in our docs.
Most users of fish are aware that they need to run bash scripts with either [bass](https://github.com/edc/bass) or a bourne-compatible shell.

Reply:
My vote is to leave this as is: we also support xonsh, nu, ion, and powershell--it's highly unlikely that we'll find syntax that works for all of them.

I think the correct way to handle this is to create a setup page as has been discussed on the v2 page, with multiple collapsible sections per-shell.

Reply:
Ok. Then you can close this PR.

Reply:
@victorhck Sorry! :grimacing: There's a lot of different concerns that go into our main documentation page due to the sheer number of shells we support--it's why we'd like to eventually move to a per-shell instruction setup.

Hope to see other contributions from you in the future!